pkg/apis/events/v1/cloudpubsubsource_defaults.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/knative-gcp/pkg/apis/duck"
+
+	"knative.dev/pkg/apis"
+	"knative.dev/pkg/ptr"
+)
+
+const (
+	defaultRetentionDuration = 7 * 24 * time.Hour
+	defaultAckDeadline       = 30 * time.Second
+)
```

Comment:
IIRC we had this in pullsubscription and you moved them to register.go
If that is the case, can you import and use those?

Reply:
The pullsubscription PR #1441 is still pending. Can we do the refactor in a following PR in the future? As the API is blocking the reconciler stuff.

Reply:
https://github.com/google/knative-gcp/pull/1441 already got merged.

Reply:
I will fix this with #1441's change tomorrow in this PR.